src/Protocol/Email.php

```diff
@@ -51,12 +51,12 @@ public static function connect(string $mailbox, string $username, string $passwo
 
 		$errors = imap_errors();
 		if (!empty($errors)) {
-			Logger::notice('IMAP Errors occured', ['errors' => $errors]);
+			Logger::notice('IMAP Errors occurred', ['errors' => $errors]);
```

Comment:
This was in the previous PR, but it was in a commit that also changed some localization items which I dropped -- I didn't properly split it out.

Reply:
It's okay, thank you for your work!